staging/src/k8s.io/apimachinery/pkg/util/proxy/upgradeaware.go

```diff
@@ -60,6 +60,8 @@ type UpgradeAwareHandler struct {
 	// Location is the location of the upstream proxy. It is used as the location to Dial on the upstream server
 	// for upgrade requests unless UseRequestLocationOnUpgrade is true.
 	Location *url.URL
+	// PrependLocationPath determines if the original path of the Location should be prepended to the upstream proxy request path
+	PrependLocationPath bool
```

Comment:
I think we want `AppendLocationPath`. Or maybe `UseLocationPath`?

Anyone have thoughts?

Reply:
Changed it to `AppendLocationPath`